stable/redis-ha/README.md

```diff
@@ -9,8 +9,8 @@ $ helm install stable/redis-ha
 ```
 
 By default this chart install 3 pods total:
- * one pod containing a redis master and sentinel containers
- * two pods each containing redis slave and sentinel containers.
+ * one pod containing a redis master, sentinel container and a prometheus metrics exporter
```

Comment:
Since this is describing what installs by 'default' then maybe something like this?
```suggestion
 * one pod containing a redis master and sentinel container (optional prometheus metrics exporter available)
```

Reply:
Done